packages/camera/camera_windows/windows/capture_controller.cpp

```diff
@@ -605,7 +606,7 @@ void CaptureControllerImpl::StartPreview() {
 void CaptureControllerImpl::StopPreview() {
   assert(capture_engine_);
 
-  if (!IsInitialized() && !preview_handler_) {
+  if (!IsInitialized() || !preview_handler_) {
```

Comment:
This was incorrect and caused a crash if you attempted to stop a preview that wasn't started.